index.html

```diff
@@ -0,0 +1,32 @@
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/styles/app.css">
+    <title>Document</title>
+</head>
+<body class="page">
+    <header class="header">
+        <img src="assets/icons/menu.svg" alt="Menu">
+    </header>
+    <section>
+        <div class="headline"> 
+            <h1 class="lines"> Hey there.</h1>
+            <h1 class="lines"> I'm THIERRY</h1>
```

Comment:
- [ ] Kindly consider applying this color `#172B4D` instead of black color following the figma design.